bindings/rust/s2n-tls/src/raw/config.rs

```diff
@@ -12,6 +12,9 @@ use std::ffi::CString;
 
 struct Owned(NonNull<s2n_config>);
 
+/// Safety: s2n_config objects can be sent across threads
+unsafe impl Send for Owned {}
```

Comment:
Wanted to point this out and make sure its true. Is it usage dependent or always true?

Reply:
It does technically depend on if the callback contexts implement `Send`. But we will require those when setting it.